From: Julius Bairaktaris <julius@bairaktaris.de>
Subject: [PATCH] nss-drv-mac80211-disable-signal-redirection

Imported from https://github.com/kuncy7/nss-packages qca-nss-drv/patches/0019-nss-drv-mac80211-disable-signal-redirection.patch (added in d7b3110); the patch file carries no author or description.
---
nss-drv-mac80211-disable-signal-redirection

Signed-off-by: Julius Bairaktaris <julius@bairaktaris.de>

--- a/nss_virt_if.c
+++ b/nss_virt_if.c
@@ -450,6 +450,8 @@ EXPORT_SYMBOL(nss_virt_if_create_sync_ne
  */
 struct nss_virt_if_handle *nss_virt_if_create_sync(struct net_device *netdev)
 {
+	if (!nss_ctl_redirect)
+	  return NULL;
 	/*
 	 * NSS_N2H_INTERFACE is the nexthop of the dynamic interface which is created for handling the
 	 * n2h traffic.
